packages/knip/test/plugins/eleventy.test.ts

```diff
@@ -1,21 +1,18 @@
 import assert from 'node:assert/strict';
 import test from 'node:test';
-import { main } from '../../src/index.js';
-import { resolve } from '../../src/util/path.js';
-import baseArguments from '../helpers/baseArguments.js';
-import baseCounters from '../helpers/baseCounters.js';
+import { default as eleventy } from '../../src/plugins/eleventy/index.js';
+import { resolve, join } from '../../src/util/path.js';
+import { getManifest, pluginConfig as config } from '../helpers/index.js';
 
 const cwd = resolve('fixtures/plugins/eleventy');
+const manifest = getManifest(cwd);
 
 test('Find dependencies in Eleventy configuration', async () => {
-  const { counters } = await main({
-    ...baseArguments,
-    cwd,
-  });
-
-  assert.deepEqual(counters, {
-    ...baseCounters,
-    processed: 2,
-    total: 2,
-  });
+  const configFilePath = join(cwd, 'eleventy.config.cjs');
+  const dependencies = await eleventy.findDependencies(configFilePath, { manifest, config });
+  assert.deepEqual(dependencies, [
+    'entry:_data/**/*.js',
+    'entry:**/*.{html,liquid,ejs,md,hbs,mustache,haml,pug,njk,11ty.js}',
+    'entry:**/*.11tydata.js',
+  ]);
```

Comment:
I think it might be good to keep the other test in there, it doesn't hurt to keep it?

Also, would be great to have a fixture, used by a test, that includes a dynamic `eleventy.config.cjs` as the one you have yourself? Then we'll cover both the default scenario and the new more dynamic one.

Reply:
Or does the name actually matter since I'm passing `configFilePath`? I could just make it like `eleventy.config.dynamic.cjs` right?

Reply:
Yes, that way you can even put multiple config files in a single fixture folder. Or create multiple fixture folders. Whatever makes sense :)